.github/workflows/push.yml

```diff
@@ -623,7 +623,7 @@ jobs:
         run: |
           ./scripts/build-docker-container.sh $DOCKER_TAG
           echo "COMMENT<<EOF" >> $GITHUB_ENV
-          printf "Affected services are: ${{needs.prepare.outputs.IMAGES}}\n" >> $GITHUB_ENV
+          printf "Affected services are: `${{needs.prepare.outputs.IMAGES}}`\n" >> $GITHUB_ENV
```

Comment:
I dont think backticks are applicable here, they are (were, deprecated now) used for command substition, e.g. `foo=$(echo "bar")`

The `${{needs.prepare.outputs.IMAGES}}` variable is a string.

Reply:
I like to use `'` for delimination in these cases

Reply:
that would end up as a literal string and would output the variable name instead of the value

Reply:
backticks are still there